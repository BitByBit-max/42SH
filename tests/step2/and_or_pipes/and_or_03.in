echo "Operators can likely be used to serve as ifs" || echo "This should not print, and is equivalent to" || echo "this second if expression." && echo "This should print, and is equivalent to" && echo "this second if expression." || echo "this second if expression."
