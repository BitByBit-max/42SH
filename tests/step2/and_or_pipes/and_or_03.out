Operators can likely be used to serve as ifs
This should print, and is equivalent to
this second if expression.
